_tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot.md

```diff
@@ -45,6 +45,15 @@ Request Field | Description
 :--- | :---
 `storage_type` | `local` indicates that all snapshot metadata and index data will be downloaded to local storage. <br /><br > `remote_snapshot` indicates that snapshot metadata will be downloaded to the cluster, but the remote repository will remain the authoritative store of the index data. Data will be downloaded and cached as necessary to service queries. At least one node in the cluster must be configured with the `search` node role in order to restore a snapshot using the `remote_snapshot` type. <br /><br > Defaults to `local`.
 
+#### Example request
+The following request restores `my-snapshot-name` as searchable snapshot.
+````json
+POST /_snapshot/my-repository-name/my-snapshot-name/_restore
+{
+  "storage_type": "remote_snapshot"
+}
+````
```

Comment:
How about including a specific index to restore?

---

The following request restores `my-index` from `my-snapshot` as a searchable snapshot.
````json
POST /_snapshot/my-repository/my-snapshot/_restore
{
  "storage_type": "remote_snapshot",
  "indices": "my-index"
}

Reply:
I was about to add it with a specific index, then I thought it might be confusing for people who are restoring the whole snapshot. But both options works for me, I don't have strong opinion for one option over the other.

Reply:
Updated by adding the my-index